config/locales/en.yml

```diff
@@ -232,3 +232,5 @@ en:
     published_version_display: 'Final Published Version'
     published_or_accepted_version_display: 'Final Published Version or Accepted Manuscript'
     no_version_display: 'None'
+    not_article_file: 'notArticleFile'
```

Comment:
Like Eric had mentioned in one of my PRs: we should stop doing this.  The `not_article_file_display` is fine, but `not_article_file` should be a constant defined in ActivityInsightOAFile.  Something like `NOT_ARTICLE_FILE = 'notArticleFile'`.  Then in `ALLOWED_VERSIONS` use `NOT_ARTICLE_FILE` instead of 'notArticleFile' and in the version_status_display method use `NOT_ARTICLE_FILE` instead of the I18n call.

Reply:
Right, I forgot about that. Thanks!